### PR TITLE
implemented disbursement tab; implemented log filtering; various bug …

### PR DESCRIPTION
**Pull request by andrewssun1**
…fixes; implemented view for log; still need to do: finish time span filter, log modal transition

### CODE REVIEW

File: src/ShoppingCart/CartQuantityChooser.js [hunk 1]
```diff
@@ -59,11 +53,19 @@ export default class CartQuantityChooser extends React.Component {
...
+                  console.log(row.original_quantity);
+                  row.quantity_cartitem = row.original_quantity;
+                }
                 );
   }
 
+  updateCart(id, Quantity){
+
+  }
+
```
Comment (pht5) [inline src/ShoppingCart/CartQuantityChooser.js]: Why does this exist?

File: src/Logs/LogTable.js [hunk 2]
```diff
@@ -2,11 +2,116 @@ var React = require('react');
...
+        ()=>{
+          this.props.cb.getRequestForLog("", ()=>{
+            this.props.cb.setState({
+                currentPage:1
+            });
+          });
+        }
+      );
+    }
+  }
```
Comment (pht5) [inline src/Logs/LogTable.js:82]: Is there an easy-ish way to refactor these into one method? If not nbd

File: src/Logs/LogTable.js [hunk 3]
```diff
@@ -2,11 +2,116 @@ var React = require('react');
...
+
+  getInitiatingUserFilter(filterHandler){
+    return <Select simpleValue value={this.state.selectedUserInitiating} placeholder="Select" options={this.state.users} onChange={this.handleSelectChangeInitiating} />
+  }
+
+  getBlah(filterHandler){
+    return(
+      <DateRangePicker></DateRangePicker>
+    );
+  }
```
Comment (pht5) [inline src/Logs/LogTable.js]: Please rename

**Review by pht5 — COMMENTED** [review]
Left a few comments below

File: src/Logs/LogTable.js (hunk 2 above)
Comment (andrewssun1) [reply src/Logs/LogTable.js:82]: wasn't easy but I did it anyway LOL

**Pull request merged**